packages/sdk/contractkit/src/test-utils/utils.ts

```diff
@@ -35,3 +39,18 @@ export const mineToNextEpoch = async (web3: Web3, epochSize: number = GANACHE_EP
   const blocksUntilNextEpoch = getFirstBlockNumberForEpoch(epochNumber + 1, epochSize) - blockNumber
   await mineBlocks(blocksUntilNextEpoch, web3)
 }
+
+export const topUpWithToken = async (
+  kit: ContractKit,
+  stableToken: StableToken,
+  account: string,
```

Comment:
when i see account i think "this is the signing/ from addres" but i think this is actually the receiver right?